addons/mail/static/src/models/activity/activity.js

```diff
@@ -283,6 +283,14 @@ function factory(dependencies) {
         creator: many2one('mail.user'),
         dateCreate: attr(),
         dateDeadline: attr(),
+        /**
+         * Backup of the feedback content of an activity to be marked as done in the popover.
+         * Feature-specific to restoring the feedback content when component is re-mounted.
+         * In all other cases, this field value should not be trusted.
+         */
+        feedbackBackup: attr({
+            default: false,
```

Comment:
It's strange to have this default at false, while it is a string. Either should be undefined or empty string I think